dpkg/config.yml

```diff
@@ -0,0 +1,60 @@
+log-type: Syslog
+keys-file: /var/lib/tox-node/keys
+udp-address: 0.0.0.0:33445
+tcp-addresses:
+  - 0.0.0.0:33445
+tcp-connections-limit: 16384
```

Comment:
It should be strictly less than `LimitNOFILE`. First off, we can have `tcp-connections-limit + 1` connections because we check the limit after a connection was accepted. Second, we open UDP socket and files which also count.

Reply:
Yep